src/test/java/com/openlattice/rehearsal/linking/LinkingFeedbackTest.kt

```diff
@@ -82,6 +82,20 @@ class LinkingFeedbackTest : SetupTestData() {
             )
 
             waitForBackgroundServices()
+
+            val linkedData = searchApi.searchEntitySetData(
+                    SearchConstraints.simpleSearchConstraints(
+                            arrayOf(linkingEntitySet.id), 0, 100, "*")).hits
+            linkingId = linkedData.map {
+                UUID.fromString(it.getValue(EdmConstants.ID_FQN).first() as String)
+            }.toSet().first()
```

Comment:
Is there a reason the entire `linkedData` list read through and mapped, instead of just doing something like 
```
linkingId = UUID.fromString(linkedData.first().getValue(EdmConstants.ID_FQN).first() as String)
```

Reply:
I just moved this code so I have not thought about this in the slightest 🤦‍♂ 🤣 

I will think about this

Reply:
Yes that makes sens

Reply:
(or just cast as a UUID) (seems cleaner)